include/mixer.h

```diff
@@ -92,7 +92,11 @@ class MixerChannel {
 	bool is_enabled = false;
 
 private:
-	MixerChannel();
+	// prevent default construction, copying, and assignment
+	MixerChannel() = delete;
```

Comment:
Just a nitpick comment; when programmer provides a parameterized constructor, C++ no longer generates a default one, so there's no need to explicitly mark it as `delete`. Not an issue though.

Reply:
Good to know! Ty :+1: